src/UI/AddSeries/SearchResultViewTemplate.hbs

```diff
@@ -80,12 +80,12 @@
                         <!--Uncomment if we need to add even more controls to add series-->
                         <!--<label style="visibility: hidden">Add</label>-->
                         <div class="btn-group">
-                            <button class="btn btn-success add x-add">
-                                <i class="icon-plus" title="Add"></i>
+                            <button class="btn btn-success add x-add" title="Add">
+                                <i class="icon-plus"></i>
                             </button>
 
-                            <button class="btn btn-success add x-add-search">
-                                <i class="icon-search" title="Add and Search for missing episodes"></i>
+                            <button class="btn btn-success add x-add-search" title="Add and Search for missing episodes">
```

Comment:
This could actually be misleading if the user has selected a different search option from the dropdown.